client/app/dashboard/appSubmit.js

```diff
@@ -1,5 +1,78 @@
-angular.module('hunt.appSubmit', [])
+angular.module('hunt.appSubmit', ['hunt.backlog'])
 
-.controller('AppSubmitController', function ($scope) {
+.controller('AppSubmitController', function ($scope, AppSubmitFactory) {
+  $scope.appSubmitInfo = {};
+  $scope.appSubmitList = [];
+
+  $scope.getAppSubmits = function () {
+    AppSubmitFactory.findAll();
```

Comment:
We'll want to add a .then and .catch to set appSubmitList equal to res.data or deal with any errors